modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/editor/configuration/FragmentEntryLinkEditorConfigContributor.java

```diff
@@ -72,6 +72,8 @@ public void populateConfigJSONObject(
 			"allowedContent", ""
 		).put(
 			"disallowedContent", "br"
+		).put(
+			"documentBrowseLinkUrl", itemSelectorURL.toString()	
```

Comment:
Is this really needed?

Reply:
Yes, we need it at the frontedn side here: 
https://github.com/liferay/alloy-editor/blob/master/src/components/buttons/button-link-edit-browse.jsx#L76

Reply:
Yes, I am aware of that.

My question was why these *browseLinkUrl's are needed if no toolbars are shown. The answer seems to be that there are not toolbars defined: 
`jsonObject.put("toolbars", JSONFactoryUtil.createJSONObject());`

I think that these urls were blind-copied in 3be60014db939f6fbd23bbf542de9a3340702066, so they can be removed.